feat: add NotificationChannel functional

adqweqd opened the pull request:
- added NotificationChannel and NotificationChannelMessage tables
- added INotificationChannelProvider and INotificationChannelMessageProvider to Dal
- added INotificationChannelService and INotificationChannelMessageService to Bll
- added NotificationChannel and NotificationChannelMessage controllers

adqweqd committed:
feat: add NotificationChannel functional
- added NotificationChannel and NotificationChannelMessage tables
- added INotificationChannelProvider and INotificationChannelMessageProvider to Dal
- added INotificationChannelService and INotificationChannelMessageService to Bll
- added NotificationChannel and NotificationChannelMessage controllers

Xpymb commented on src/backend/dotnet/Services/Portal/Freaks.Portal.WebApi/Controllers/Notification/NotificationChannelMessageController.cs:
```diff
@@ -0,0 +1,56 @@
+        return await _service.GetListAsync(request);
+    }
+    
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <param name="channelId"></param>
+    /// <param name="request"></param>
+    /// <returns></returns>
+    [HttpPost]
+    public async Task<NotificationChannelMessageDto> CreateAsync(
+                 [FromRoute] int channelId,
+                 [FromBody] CreateNotificationChannelMessageRequest request)
+    {
+        var result = await _service.CreateAsync(channelId, request);
+        return result;
+    }
```
этого в контроллере не нужно, создаваться они будут через рэббит

Xpymb reviewed (changes requested):
Нужно сделать, перед мёрджем:
1) Расписать саммари, которые остались пустыми
2) Добавить отправку ивентов в ws, пример: 
https://github.com/Xpymb/freaks_aa/blob/develop/src/backend/dotnet/Services/Portal/Bll/Freaks.Portal.Bll.Implementation/RaidSummary/RaidLootService.cs#L101

adqweqd committed:
feat: add WS in services and corrected comments
-added WS in NotificationChannelMessageService and NotificationChannelService
-updated summary
-removed the create method in the NotificationChannelMessageController

adqweqd commented:
@Xpymb Поправил замечания

Xpymb commented on src/backend/dotnet/Services/Portal/Bll/Freaks.Portal.Bll.Implementation/Notification/NotificationChannelService.cs:
```diff
@@ -7,24 +10,27 @@
 
 namespace Freaks.Portal.Bll.Implementation.Notification;
 /// <summary>
```
отступ

Xpymb commented on src/backend/dotnet/Services/Portal/Bll/Freaks.Portal.Bll.Implementation/Notification/NotificationChannelMessageService.cs:
```diff
@@ -10,56 +12,80 @@
-    /// <param name="request"></param>
-    /// <returns></returns>
+    
     public async Task<PaginatedList<NotificationChannelMessageDto>> GetListAsync(GetNotificationChannelMessageRequest request)
```
пропустил саммари

adqweqd committed:
feat: some fixes by code style
